metabrainz/templates/index/privacy.html

```diff
@@ -7,12 +7,144 @@ <h1 class="page-title">{{ _('Privacy Policy') }}</h1>
+    made available to the public via our site usage pages.
+  </p>
+
+  <h3>{{ _('Third-Party content') }}</h3>
+  <p>
+    Our project's web pages may load some third-party content. Some, but possibly not all sites are listed below:
+  </p>
+
+  <ol>
+    <li>
+      Cover art images: These are provided by the various cover art sites that have given MusicBrainz permission to do so. This means 
+      that the relevant site will know (if they want to) your IP address, User-Agent string, etc., and which MusicBrainz web page you 
+      were visiting (the one which included the cover art image).
+    </li>
+    <li>
+      Google Analytics: We use Google Analytics in our web pages to get a picture of who visits MusicBrainz. From these analytics we 
+      can set targets for supported browser versions, screen sizes and other useful information that we take into consideration when 
+      developing or improving features on our site. If you object to the use of Google Analytics, please consider blocking it in your 
+      browser.
+    </li>
+    <li>
+      AcoustID: Our fingerprints tab loads fingerprint information from acoustid.org.
+    </li>
+    <li>
+      reCAPTCHA: Our account creation page loads third party content in order to prevent spammers from creating accounts.
+    </li>
+  </ol>
+
+  <p>
+    The above information assumes that you are using "normal" web browser settings, whereby images are always loaded and HTTP referrer 
+    information is always sent.
+  </p>
+
+  <p>
+    It is impractical to constantly modify this privacy policy when MusicBrainz changes which third party content it utilizes. We will 
+    aim to use common sense, respect to our community and community review when adding more content from third party sites. We will also 
+    periodically review our policy to ensure that it remains current.
+  </p>
+
+  <h2>{{ _('Applicable to account holders') }}</h2>
+
+  <h3>{{ _('Account creation') }}</h3>
+  <p>
+    When creating an account with a MetaBrainz project you need to pick a unique username and choose a password, the username you pick is the 
+    only name associated with your account and will be what other MetaBrainz users know you as. You may optionally also tell us other 
+    information about yourself that will be displayed to other MetaBrainz users and the public.
+  </p>
+
+  <p>
+    Additionally, in order to edit the databases you will need to provide a confirmed email address. This email address will be held in 
+    confidence, the only method of revealing your email address to another user is if you choose to send a message to another 
+    user and enable the option to "reveal my email address". Changing the email address stored with your account will require you to 
+    verify the new address.
+  </p>
+
+  <h3>{{ _('Edits and Notes') }}</h3>
+  <p>
+    If you make any changes to the a MetaBrainz project database, such as adding any data (including fingerprint submissions), the details 
```

Comment:
1. `s/the a/a/`
2. I'd remove the fingerprint submissions note, as fingerprint submissions are no longer MB edits (and doesn't even need a MusicBrainz/MetaBrainz account).

Reply:
@mayhem And this?